packages/office-ui-fabric-react/src/components/Icon/examples/Icon.Basic.Example.tsx

```diff
@@ -14,8 +14,8 @@ export const IconBasicExample: React.FunctionComponent = () => {
   // You could also use the standard Icon here.
   return (
     <div>
-      <FontIcon iconName="CompassNW" className={iconClass} />
-      <FontIcon iconName="Dictionary" className={iconClass} />
+      <FontIcon iconName="CompassNW" aria-label="compass" className={iconClass} />
```

Comment:
I'm not sure we should add labels to the examples in all cases. Ideally we should find some way of demonstrating that you add an aria-label when the icon has additional meaning which isn't conveyed by adjacent text (and don't necessarily add it if the icon is just presentational).

Reply:
I'm happy to remove it, or open a separate issue/PR about examples and the Do's/Don'ts (I'd love to see a line in there about labeling, if possible 😃).

Is there a way you'd prefer to handle it? Just based on the examples themselves, the icons seem to require a label (they appear neither presentational nor accompanied by text), so that's why I added labels on a couple of them. I'm totally happy to remove/change/open a different PR/do whatever you think best, though.

Reply:
I see what you mean. The examples are a bit unfortunate because they're not the most realistic demonstration of how to use icons. So the ideal would be to switch to still-simple yet more realistic examples (if we can come up with some), but the (probably) easier option would be to leave the labels and add a comment in each example about when you should actually use labels or not.